src/common/closest-parent/index.js

```diff
@@ -0,0 +1,32 @@
+// https://developer.mozilla.org/en-US/docs/Web/API/Element/matches
+
+function closestParent(el, selector) {
+    let elem = el;
+
+    // Element.matches() polyfill
```

Comment:
Do we need this polyfill? Browser support seems solid.

Reply:
We could certainly entertain not doing the polyfill. I could shorten it to this:
```
if (!Element.prototype.matches) {
  Element.prototype.matches = Element.prototype.msMatchesSelector ||
                              Element.prototype.webkitMatchesSelector;
}
```

Reply:
Your call. I just looked at MDN and CanIUse and it seemed well supported. I see now though that IE11 only has partial support.